### ISSUE TITLE
Add create element helper functions

### ISSUE THREAD

**Issue by alexwkleung** [alexwkleung/Iris#62]
Iris strictly uses dynamically created elements (via `Document.createElement()` and the related) but everything is done manually without any helper functions. This introduces extreme verbosity and code that is hard to manage.

In the core cleanup, I want to simplify how I create elements in the front-end. Therefore, introducing helper functions to do the heavy lifting would be nice. 

The JSX transform output seems quite interesting, so I plan on taking inspiration from that to design the helper function.

JSX transform output example (https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html):

```ts
function App() {
  return _jsx('h1', { children: 'Hello world' });
}
```  

Basically trying to avoid using React or TypeScript's JSX compiler option and simply create my own function to wrap around `Document.createElement()` and the related. I don't want to end up creating a Virtual DOM from scratch as that is not my goal here. Just keeping everything vanilla as possible.
